tests/python_package_test/test_dask.py

```diff
@@ -1255,3 +1255,59 @@ def test_sklearn_integration(estimator, check, client):
 def test_parameters_default_constructible(estimator):
     name, Estimator = estimator.__class__.__name__, estimator.__class__
     sklearn_checks.check_parameters_default_constructible(name, Estimator)
+
+
+@pytest.mark.parametrize('task', ['binary_classification', 'multi-class_classification', 'regression'])
+@pytest.mark.parametrize('output', data_output)
+def test_predict_with_raw_score(task, output, client):
+    if task.endswith('classification'):
+        objective = 'classification'
```

Comment:
please change this to match the other tests, where `task` is passed directly into `_create_data()`: https://github.com/microsoft/LightGBM/blob/6356e659af74909aa77eb87d54b79f0373772509/tests/python_package_test/test_dask.py#L641.

Creating another layer of indirection here is unnecessary and could lead to mistakes. The names you use for `task` and the names accepted by the `objective` argument of `_create_data()` should exactly match.

If you agree with my proposal in another comment to add a `multiclass-classification` task, I think it would be appropriate in that PR to add this in `_create_data()`:

```python
if object == 'multiclass-classification' and centers < 3:
    centers = 3
```

Reply:
I agree the data creation for this test is pretty annoying. Should I open that PR modifying the tasks and update this when that gets merged?

Reply:
yes please! That would be great. I think that one will be non-controversial and should be a quick review.